src/sensor/ping360.h

```diff
@@ -311,6 +314,28 @@ class Ping360 : public PingSensor
     }
     Q_PROPERTY(int number_of_points READ number_of_points WRITE set_number_of_points NOTIFY numberOfPointsChanged)
 
+    /**
+     * @brief Return sector size in gradians
+     *
+     * @return int
+     */
+    int sectorSize() { return _sectorSize; }
```

Comment:
Suggestion for the future: rather than using properties for 'central angle' and 'sector size', I think it _might_ be easier to handle all of this math with ping properties of 'scan start angle' and 'scan end angle'.